pkg/util/ecs/metadata/detection.go

```diff
@@ -68,6 +69,10 @@ func detectAgentV1URL() (string, error) {
 func getAgentV1ContainerURLs(ctx context.Context) ([]string, error) {
 	var urls []string
 
+	if !config.IsFeaturePresent(config.Docker) {
+		return nil, errors.New("docker feature not activated")
```

Comment:
For consistency 
```suggestion
		return nil, errors.New("Docker feature not activated")
```

Reply:
`Error strings should not be capitalized (unless beginning with **proper nouns or acronyms**)` 😅  
[source](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)

Reply:
Works for me 👍